chipsec/options/cmd_options.ini

```diff
@@ -9,3 +9,6 @@ deltas_output_file = log_output_deltas.json
 smbios_get_type = raw
 #smbios_get_type = decoded
 #default_helper = replayhelper
+
+[Exclude]
+#modules = common.bios_wp,common.cpu.cpu_info

```

Comment:
This should go under [Main_Config] since it's being consumed by chipsec_main.

Reply:
That depends.  I originally had it there but chose this way because this list can be excessive and multiline.  I'll reevaluate it.

Reply:
I think that's fine to be long and excessive while being in Main_Config section. The section defines what consumes it.